components/status/HeatMapColorBar.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div>
+    <div id="parent-container">
+      <div
+        id="startup-heatmap"
+        class="div__heatmap-gradient-holder"
+        :style="value"
+      ></div>
+    </div>
+    <!-- prettier-ignore -->
```

Comment:
it looks like these comments aren't doing anything. can they be removed?

Reply:
Done

Reply:
After following the comments I got the below error:-

/home/ubuntu/environment/mantarray-frontend-components/components/plate_based_widgets/mapeditor/PlateMapEditor.vue
   13:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
   34:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
   55:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
   76:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
   97:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
  118:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
  139:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
  160:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
  181:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error
  202:9  error  Parsing error: abrupt-closing-of-empty-comment  vue/no-parsing-error

✖ 10 problems (10 errors, 0 warnings)

So reverting back to <!-- prettier-ignore -->

Reply:
Fixing quite a few other stuff.